toolkit/tools/imagecustomizerapi/systemconfig.go

```diff
@@ -21,6 +22,7 @@ type SystemConfig struct {
 	PackageListsUpdate      []string                  `yaml:"PackageListsUpdate"`
 	PackagesUpdate          []string                  `yaml:"PackagesUpdate"`
```

Comment:
There is supposed to be an EnableGrubMkconfig param either 1. already introduced three days ago, see https://github.com/microsoft/CBL-Mariner/commit/685a93c2bb0f1ab8dd124ccdf050704ac1c086d6#diff-322f8615bc66a1032691d83e59eacc865d47935978e7bfd07faeb37299e984ddR23    2. cherry-picked in here

I think you might need to rebase against the head of 3.0-dev to clear this up?

Reply:
I think this is all in the branch already: https://github.com/dmcilvaney/CBL-Mariner/blob/371a4e6979e106218ecf012ccfb78595a601a0f5/toolkit/tools/imagegen/configuration/systemconfig.go#L23